packages/node-build-scripts/src/__tests__/cssVariables.test.ts

```diff
@@ -4,11 +4,15 @@
 
 import { describe, expect, test } from "@jest/globals";
 import { readFileSync } from "node:fs";
-import { join, resolve } from "node:path";
+import { join, resolve, dirname } from "node:path";
+import { fileURLToPath } from "node:url";
 
 import { generateLessVariables, generateScssVariables, getParsedVars } from "../cssVariables.mjs";
 
-const FIXTURES_DIR = join(import.meta.dirname, "__fixtures__");
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
```

Comment:
Does `__dirname` need to be defined? Curious if this works without it?

https://nodejs.org/docs/latest/api/modules.html#__dirname

Reply:
Yeah, this was the first thing I tried but it sadly does not seem to work:

![Screenshot 2024-08-05 at 17 15 05@2x](https://github.com/user-attachments/assets/fcc44ded-0150-48a1-871e-70a7052d29d7)

Reply:
Oh I see. It looks like that's because we're already opting into Jest's ESM support based on the `node:80071` warning. It makes sense that `__dirname` (a CommonJS feature) isn't available given that. Thanks for trying.